### DISCUSSION TITLE
Changes to Master schema.

### DISCUSSION THREAD

**Discussion by NTDLS** [NTDLS/Katzebase#95] [Announcements]
@ingted 
I added a field to Master:Account and created new schemas "Master:Role" and "Master:Membership", this schema change **will cause your login to fail**, so I also added a "magic user" when running in debug mode that will allow you to login and drop the master schema.

- Login with username: ```debug```
- `DROP SCHEMA Master` (you might have to run the drop statement twice).
- Stop and start the server. The missing Master schema will be re-created with the default username and password.

**Comment by ingted** [comment]
Roger!!!

**Comment by NTDLS** [comment]
Finally got the engine to a point where it can perform tasks with scripts, so creation of the Master schema has been moved to:
[Katzebase.Engine\Scripts\CreateMasterSchema.kbs](https://github.com/NTDLS/Katzebase/blob/master/NTDLS.Katzebase.Engine/Scripts/CreateMasterSchema.kbs)

These scripts are added to the project as "embedded resources" and can be executed from within the engine to perform tasks such as user creation and role modification.

Some examples: [Katzebase.Engine\Interactions\Management\PolicyManager.cs](https://github.com/NTDLS/Katzebase/blob/master/NTDLS.Katzebase.Engine/Interactions/Management/PolicyManager.cs)
